### ISSUE TITLE
Not making use of all data in test set

### ISSUE THREAD

**Issue by gorold** [sanatonek/TNC_representation_learning#1]
https://github.com/sanatonek/TNC_representation_learning/blob/7f5ac2dc99c412cb5cb9266793ec3f20809a8911/evaluations/classification_test.py#L160-L172

Hi Sana, would like to inform you of a potential issue in the testing code. The number of points being used from the test set is limited to the length of the training set. This could lead to some missing class labels in certain datasets.

**Comment by 943fansi** [comment]
@gorold Hello, Gerald Woo! I am very interested in your work. When will you open the complete code of ICLR 2022 Paper `COST: CONTRASTIVE LEARNING OF DISENTANGLED SEASONAL-TREND REPRESENTATIONS FOR TIME SERIES FORECASTING`?

**Comment by gorold** [comment]
Hi @943fansi, thanks for your interest, perhaps you can drop me an email and I will get back to you from there, since this is not related to the TNC repo or this issue.
